Common/Securities/Volatility/IVolatilityModel.cs

```diff
@@ -34,6 +35,14 @@ public interface IVolatilityModel
         /// <param name="security">The security to calculate volatility for</param>
         /// <param name="data">The new data used to update the model</param>
         void Update(Security security, BaseData data);
+
+        /// <summary>
+        /// Returns history requirements for the volatility model expressed in the form of history request
+        /// </summary>
+        /// <param name="security">The security of the request</param>
+        /// <param name="utcTime">The date/time of the request</param>
+        /// <returns>History request object, or null if no requirements</returns>
+        HistoryRequest HistoryRequirements(Security security, DateTime utcTime);
```

Comment:
A plus side to returning `IEnumerable<HistoryRequest` is we wouldn't need to return `null` in the `NullVolatilityModel` but rather just an empty enumerable, which would preclude the `AlgorithmManager` from directly checking if the vol model is the `NullVolatilityModel`. 

Ideally, when using the [null pattern](https://en.wikipedia.org/wiki/Null_Object_pattern), no consumer of the object needs to know that it is a special implementation